archaius2-archaius1-bridge/src/main/java/com/netflix/archaius/bridge/Archaius1DelegatingConfig.java

```diff
@@ -0,0 +1,74 @@
+package com.netflix.archaius.bridge;
+
+import java.util.Iterator;
+import java.util.Properties;
+
+import org.apache.commons.configuration.AbstractConfiguration;
+import org.apache.commons.configuration.event.ConfigurationEvent;
+import org.apache.commons.configuration.event.ConfigurationListener;
+
+import com.netflix.archaius.api.Config;
+import com.netflix.archaius.api.config.SettableConfig;
+import com.netflix.archaius.config.AbstractConfig;
+import com.netflix.config.ConfigurationManager;
+
+public class Archaius1DelegatingConfig extends AbstractConfig implements SettableConfig {
+    
+    private AbstractConfiguration archaius1Config;
+
+    public Archaius1DelegatingConfig() {
+        archaius1Config = ConfigurationManager.getConfigInstance();
```

Comment:
Would be good to double check with Howard/Eran, but just want to make sure there are no static init implications with accidentally initializing ConfigurationManager for the first time here.